oracle/README.md

```diff
@@ -37,31 +37,57 @@ oracle/
 
 TODO: need UI for this to be relevant
 
-### Node
-In order to run the Oracle node make sure you are in the root of this project i.e `/path/to/oracle/<you are here>`
+The project can be started by executing the following steps:
```

Comment:
This needs a heading since it looks like it all falls under the UI section defined above